packages/financial-templates-lib/test/price-feed/InsuredBridgePriceFeed.js

```diff
@@ -13,6 +13,10 @@ const { deployOptimismContractMock } = require("../../../core/test/insured-bridg
 const winston = require("winston");
 const { assert } = require("chai");
 
+// Note: This bot should not be opinionated about whether it is connected to Optimism or Arbitrum, so we'll default to
```

Comment:
yup! Ideally each is tested. can add more tests later.

Reply:
I addressed this comment and removed all assumptions about which network we're testing